### ISSUE TITLE
Set path in opts to upload, without path random.

### ISSUE THREAD

**Issue by nicholasess** [Turistforeningen/node-s3-uploader#36]
I need to set the after bucket, for example:

bucket/mypath/ap-small.jpg

**Comment by Starefossen** [comment]
Hi @nicholasess, thanks for suggesting this feature. This has been implemented in #47 and landed `s3-uploader@1.1.0-rc.2` please give it a try.

```js
var client = new Uploader('my-s3-bucket');

client.upload('/my/awsome/image.jpg', {
  path: 'custom/your/path/new_filename'
});
```

**Comment by nicholasess** [comment]
Thanks @Starefossen I'll try the feature.
